strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-nuget-layout-provider/src/main/java/org/carlspring/strongbox/artifact/coordinates/PathNupkg.java

```diff
@@ -110,6 +133,10 @@ public NugetArtifactCoordinates readArtifactCooridnates()
     @Override
```

Comment:
Cosmetic: Something's wrong with your IDE-s indentation settings. I know you re-installed recently. Would you mind using the Eclipse style, as described [here](https://github.com/strongbox/strongbox/wiki/Coding-Convention).

Reply:
I have reformatted this class with my old styles I have used always

Reply:
Interesting... Well, if you check just a few lines below, you'll see that the code block you changed is indented by on less character. Check lines 136-164. I've seen this in a number of places across the pull. It's not a massive issue, but it's not compliant with the coding convention. Would you mind taking another look?

Reply:
I have just pushed my commit :)